grammars/python.cson

```diff
@@ -1151,7 +1151,7 @@
         'name': 'string.quoted.double.block.sql.python'
         'patterns': [
           {
-             'begin': '(?=\\s*(SELECT|INSERT|UPDATE|DELETE|CREATE|REPLACE|ALTER))'
+             'begin': '(?=\\s*(SELECT|INSERT|UPDATE|DELETE|CREATE|REPLACE|ALTER|WITH))'
```

Comment:
I only count three changes here. There should be four:
* `'name': 'string.quoted.double.block.sql.python'`
* `'name': 'string.quoted.single.block.sql.python'`
* `'name': 'string.quoted.double.single-line.sql.python'`
* `'name': 'string.quoted.single.single-line.python'`

Maybe add tests to make sure you cover all four?